app/rtc_compare.py

```diff
@@ -0,0 +1,386 @@
+'''
+Compare two RTC products if they are equivalent
+Part of the codes are copied from PROTEUS SAS
+'''
+
+import argparse
+import itertools
+
+import h5py
+import numpy as np
+
+RTC_S1_PRODUCTS_ERROR_TOLERANCE = 1e-6
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='Compare two RTC products',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # Inputs
+    parser.add_argument('input_file',
+                        type=str,
+                        nargs=2,
+                        help='Input RTC products in NETCDF/HDF5 format')
+
+    return parser
+
+def printout_data_difference(val_1, val_2, indent=4):
```

Comment:
```suggestion
def print_data_difference(val_1, val_2, indent=4):
```
If you prefer to keep using "printout", maybe you can add an underscore in between:
 
```def print_out_data_difference(val_1, val_2, indent=4): ```